code/style.css

```diff
@@ -65,12 +65,13 @@ select {
   height: 165px;
   overflow: hidden;
   border: 3px solid var(--secondary);
-  border-radius: 4px;
-  margin: 3px;
+  border-radius: 10px;
+  margin: 5px;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: space-between;
+  transition: 1.5s;
```

Comment:
Great that you've used some kind of animation transition, looks nice on hovering!